posthog/queries/session_recordings/session_recording_list.py

```diff
@@ -156,8 +157,10 @@ def _determine_should_join_events(self):
 
     def _get_properties_select_clause(self) -> str:
         session_id_clause, _ = get_property_string_expr("events", "$session_id", "'$session_id'", "properties")
+        window_id_clause, _ = get_property_string_expr("events", "$window_id", "'$window_id'", "properties")
```

Comment:
I think we can get rid of these because we now have a migration that guarantees these two columns are materialized out.

Reply:
nice!